core/players-handbook-2024/races/race-goliath.xml

```diff
@@ -0,0 +1,149 @@
+		</description>
+		<sheet alt="Cloud’s Jaunt" action="Bonus Action" usage="{{proficiency}}/Long Rest">
+			<description>You magically teleport up to 30 feet to an unoccu­pied space you can see.</description>
+		</sheet>
+	</element>
+	<element name="Fire’s Burn (Fire Giant)" type="Racial Trait" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_RACIAL_TRAIT_GOLIATH_GIANT_ANCESTRY_FIRES_BURN">
+		<supports>Giant Ancestry 2024</supports>
+		<description>
+			<p>When you hit a target with an attack roll and deal damage to it, you can also deal ldlO Fire damage to that target.</p> 
+		</description>
+		<sheet alt="Fire’s Burn" usage="{{proficiency}}/Long Rest">
+			<description>When you hit a target with an attack roll and deal damage to it, you can also deal ldlO Fire damage to that target.</description>
+		</sheet>
+	</element>
+	<element name="Frost’s Chill (Frost Giant)" type="Racial Trait" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_RACIAL_TRAIT_GOLIATH_GIANT_ANCESTRY_FROSTS_CHILL">
+		<supports>Giant Ancestry 2024</supports>
+		<description>
+			<p>When you hit a target with an attack roll and deal damage to it, you can also deal ld6 Cold damage to that target and reduce its Speed by 10 feet until the start of your next turn.</p> 
+		</description>
+		<sheet alt="Frost’s Chill" usage="{{proficiency}}/Long Rest">
+			<description>When you hit a target with an attack roll and deal damage to it, you can also deal ld6 Cold damage to that target and reduce its Speed by 10 feet until the start of your next turn.</description>
+		</sheet>
+	</element>
+	<element name="Hill’s Tumble (Hill Giant)" type="Racial Trait" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_RACIAL_TRAIT_GOLIATH_GIANT_ANCESTRY_HILLS_TUMBLE">
+		<supports>Giant Ancestry 2024</supports>
+		<description>
+			<p>When you hit a Large or smaller creature with an attack roll and deal damage to it, you can give that target the Prone condition.</p> 
+		</description>
+		<sheet alt="Hill’s Tumble" usage="{{proficiency}}/Long Rest">
+			<description>When you hit a Large or smaller creature with an attack roll and deal damage to it, you can give that target the Prone condition.</description>
+		</sheet>
+	</element>
+	<element name="Stone’s Endurance (Stone Giant)" type="Racial Trait" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_RACIAL_TRAIT_GOLIATH_GIANT_ANCESTRY_STONES_ENDURANCE">
+		<supports>Giant Ancestry 2024</supports>
+		<description>
+			<p>When you take damage, you can take a Reaction to roll ldl2. Add your Constitution modifier to the number rolled and reduce the damage by that total.</p> 
+		</description>
+		<sheet alt="Stone’s Endurance" action="Reaction" usage="{{proficiency}}/Long Rest">
+			<description>When you take damage, you can take a Reaction to roll ldl2. Add your Constitution modifier to the number rolled and reduce the damage by that total.</description>
```

Comment:
seems like anywhere 1d# was copied from your source, it translated it into letters instead of numbers

```suggestion
		<description>
			<p>When you take damage, you can take a Reaction to roll 1d12. Add your Constitution modifier to the number rolled and reduce the damage by that total.</p> 
		</description>
		<sheet alt="Stone’s Endurance" action="Reaction" usage="{{proficiency}}/Long Rest">
			<description>When you take damage, you can reduce the damage by 1d12+{{constitution:modifier}}</description>
```